orchestration/src/main/java/com/sap/ai/sdk/orchestration/AssistantMessage.java

```diff
@@ -0,0 +1,11 @@
+package com.sap.ai.sdk.orchestration;
+
+import com.sap.ai.sdk.orchestration.client.model.ChatMessage;
+import javax.annotation.Nonnull;
+
+public class AssistantMessage extends ChatMessage {
```

Comment:
(Major)

I think we aligned on not-using inheritance between convenience- and low-level API.

For example see `OrchestrationAiModel`.

Please correct me if my observation / conclusion is wrong, @MatKuhr

Reply:
Yes, it's safer to clearly separate the two APIs